library/src/main/java/net/yslibrary/licenseadapter/internal/LibrariesHolder.java

```diff
@@ -46,7 +46,7 @@ public void run() {
             cachedLicenses.put(library, license);
             notify(license, null);
           } catch (Exception e) {
-            notify(null, e);
+            notify(library.getLicense(), e);
```

Comment:
Why change this? That breaks the xor contract for nullability and doesn't change any behavior as far as I can tell.